src/pages/ProfilePage.tsx

```diff
@@ -0,0 +1,65 @@
+import Sidebar from "../components/Sidebar";
+import "../index.css";
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+const ProfilePage = (): JSX.Element => {
+    return (
+        <div className="container">
+            <Sidebar />
+            <div className='some-page-wrapper'>
+                <div className="main-content">
+                    <h1 className="content-title">Profile</h1>
+                </div>
+                <div className='column outset'>
+
+                    <div className='one-column inset'>
+                        <h1 className="second-text">Personal</h1>
+                        <form>
+                            <input type="text" placeholder="First Name" className="text-input" />
+                        </form>
+
+                        <form>
+                            <input type="text" placeholder="Last name" className="text-input" />
+                        </form>
+                        <form>
+                            <input type="text" placeholder="Phone" className="text-input" />
+                        </form>
+                        <form>
+                            <input type="text" placeholder="Gender" className="text-input" />
+                        </form>
+                        <form>
+                            <input type="text" placeholder="Pronouns" className="text-input" />
+                        </form>
+                        <form>
+                            <input type="text" placeholder="Ethnicity" className="text-input" />
+                        </form>
+                    </div>
+
+
+                    <div className='one-column inset'>
+                        <h1 className="second-text">Education</h1>
+                        <form>
```

Comment:
And here, just one form tag is good for all of these inputs. Apply inset class to all inputs as well